x-pack/plugins/alerts/server/config.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema, TypeOf } from '@kbn/config-schema';
+import { validateDurationSchema } from './lib';
+
+export const configSchema = schema.object({
+  invalidateApiKeysTask: schema.object({
+    interval: schema.string({ validate: validateDurationSchema, defaultValue: '5m' }),
```

Comment:
👍 I like the `5m` as I believe this reflects the default task timeout value 🙂

Reply:
>but does that mean we'd guarantee a failure on the next retry if a task does timeout? I think we'd retrieve the key again, but not sure.

It will retrieve the key again at the next execution. Thinking of it, actions also use the execution's API key to run so a longer gap may not be a bad idea.

Reply:
Friendly reminder to make sure these settings are allowed in the docker config and ESS if we intend for them to be configured by end users (which I assume we do if they're part of the config schema)

Reply:
@YulNaumenko is a PR coming to allow-list these new settings on cloud? I'm going to open a discuss issue on if we should increase the `removalDelay`.

Reply:
> @YulNaumenko is a PR coming to allow-list these new settings on cloud? I'm going to open a discuss issue on if we should increase the `removalDelay`.

Yes, we need this config props, and healthCheck interval. Forgot about that - I will open an issue.